### DISCUSSION TITLE
Hello, which .deb have I to download for my Raspberry 4? Thank you

### DISCUSSION THREAD

**Discussion by pdesimonee** [Motion-Project/motion#1468] [Q&A]
0. Reviewed guide and contributing documents? (Yes/No):
1. version [x.y.z, hash, other]: 
2. installed as a package or compiled from sources [deb, rpm, git, other]: 
3. standalone or part of third party [motion, MotionEyeOS, other]: 
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: 
5. hardware [x86, ARM, other]: 
6. operating system [16.04, Stretch, etc, FreeBSD, other]:

**Comment by pdesimonee** [comment]
well, MotioneyeOS is dead and I would like to put in place my stuff using directly motion.

**Comment by Mr-Dave** [comment]
You will probably need to build it from source to get version 4.4.0.  The deb packages in the release page only have buster and the latest version is bullseye.  

The following [page](https://www.youtube.com/channel/UCZpOjUMnc-uO3qIyZAD_4lg/videos) has a couple of ~10 minute videos that show how to build from source or create your own deb.
